package.json

```diff
@@ -628,9 +637,14 @@
                     "group": "1@1"
                 },
                 {
-                    "command": "appService.RemoveCosmosDBConnection",
+                    "command": "appService.RevealConnectionInAppSettings",
                     "when": "view == azureAppService && viewItem == cosmosDBConnection",
                     "group": "2@1"
```

Comment:
I feel like both reveal commands should be in the same "group". Reveal in Cosmos should be "1@1" and reveal in app settings should be "1@2". Please comment with a screenshot after you're done